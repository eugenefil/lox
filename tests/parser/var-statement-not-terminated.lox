var x = 5 _
