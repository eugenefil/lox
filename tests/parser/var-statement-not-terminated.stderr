error: expected ';'
 --> $DIR/var-statement-not-terminated.lox:1:11
  |
1 | var x = 5 _
  |           ^
